lib/classes/Utils.js

```diff
@@ -40,7 +40,8 @@ class Utils {
       contents = JSON.stringify(contents, null, 2);
     }
 
-    if (filePath.indexOf('.yaml') !== -1 && typeof contents !== 'string') {
+    if (((filePath.indexOf('.yaml') !== -1) || (filePath.indexOf('.yml') !== -1))
+      && typeof contents !== 'string') {
```

Comment:
this if expression is getting complex, took me couple of seconds to figure out what's going on there. I think it'd cleaner if it's something like:

```js
const yamlFileExists = (filePath.indexOf('.yaml') !== -1);
const ymlFileExists = (filePath.indexOf('.yml') !== -1);

if ((yamlFileExists || ymlFileExists) && typeof contents !== 'string') {
  ...
}
```
Haven't really tested this snippet, but I think something like that is cleaner and will help us debug quicker in the future.